Source/ExecutionEngine/CommandLineOptions.cs

```diff
@@ -108,6 +108,17 @@ public string FileTimestamp
       }
     }
 
+    public void ExpandFilename(ref string pattern, string logPrefix, string fileTimestamp)
+    {
+      if (pattern != null)
+      {
+        pattern = pattern.Replace("@PREFIX@", logPrefix).Replace("@TIME@", fileTimestamp);
+        string fn = Files.Count == 0 ? "" : Files[Files.Count - 1];
+        fn = Util.EscapeFilename(fn);
+        pattern = pattern.Replace("@FILE@", fn);
+      }
+    }
+
```

Comment:
Is this version of `ExpandFilename` used anywhere?

Reply:
Only in Boogie consumers